sdk/aqueduct/backend/response_models.py

```diff
@@ -231,10 +231,7 @@ class SavedObjectUpdate(BaseModel):
     operator_name: str
     modified_at: str
     integration_name: str
-    integration_id: uuid.UUID
-    service: ServiceType
-    object_name: str
-    update_mode: str
+    spec: LoadSpec
```

Comment:
Does this need to be `Optional`? I noticed the go struct is a pointer

Reply:
Yeah good catch! Since the load spec must exist, I just made the field on a go struct a value instead of pointer.